README.md

```diff
@@ -64,56 +75,68 @@ APOLLO_KEY=<enter copied key here>
 
 After this is done, VS Code can be reloaded and the Apollo integration will connect to GraphOS Studio to provide autocomplete, validation, and more.
 
-<h3 id="local-schemas">Local schemas</h3>
+</details>
+
+<details>
+<summary>
+<h3 id="local-schemas">Setting the Extension up for use with a local schema</h3>
+</summary>
 
-Sometimes it may make sense to link the editor to a locally running version of a schema to try out new designs that are in active development. To do this, the `apollo.config.js` file can be linked to a local service definition:
+Sometimes it may make sense to link the editor to a locally running version of a schema to try out new designs that are in active development. To do this, the `apollo.config.json` file can be linked to a local service definition:
 
-```js
-module.exports = {
-  client: {
-    service: {
-      name: "my-graphql-app",
-      url: "http://localhost:4000/graphql",
-    },
-  },
-};
+```json
+{
+  "client": {
+    "service": {
+      "name": "my-graphql-app",
+      "url": "http://localhost:4000/graphql"
+    }
+  }
+}
 ```
 
 Linking to the local schema won't provide all features, such as switching graph variants and performance metrics.
 
-<h3 id="local-schema-files">Local schema files</h3>
+</details>
+
+<details>
+<summary>
```

Comment:
I would include this under the same header (Configure extension for local schemas) and present two options at the beginning:

- Use introspection of a locally running service
- Use with a local schema file.

IMO, since these are two flavors of the "Configure extension for local schemas" they should be separate `<h4>`s within that collapsible section.

Reply:
People have been missing these options a lot, so I want to have both options as visible as possible. It's really pulling down our marketplace ratings.